irispy/data/_sample.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""IrisPy Sample and test files"""
+
+import os.path
+import socket
+import warnings
+from zipfile import ZipFile
+from shutil import move
+
+from astropy.utils.data import download_file
+
+from sunpy.util.net import url_exists
+from sunpy.util.config import get_and_create_sample_dir
+from sunpy import config
+
+__author__ = "Steven Christe"
+__email__ = "steven.christe@nasa.gov"
+
+# Creating the directory for sample files to be downloaded
+sampledata_dir = get_and_create_sample_dir()
+
+# urls to search for the data
+_base_urls = (
+    'http://data.sunpy.org/sample-data/',
+    'http://hesperia.gsfc.nasa.gov/~schriste/sunpy-sample-data/')
+
+_files = {"SJI_CUBE_2832": ("iris_l2_20170502_052551_3893010094_SJI_2832_t000.fits", ""),
+          "SJI_CUBE_1330": ("iris_l2_20170502_052551_3893010094_SJI_1330_t000.fits", ""),
+          "SJI_CUBE_2796": ("iris_l2_20170502_052551_3893010094_SJI_2796_t000.fits", ""),
+          "SJI_CUBE_1400": ("iris_l2_20170502_052551_3893010094_SJI_1400_t000.fits", ""),
+          "RASTER": ("iris_l2_20170502_052551_3893010094_raster_t000_r00000.fits", ".tar.gz")
+         }
+
+sample_files = {}
+for key in _files:
+    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
+
+
+def download_data(progress=True, overwrite=True, timeout=None):
+    """
+    Download the sample/test data.
+    Parameters
+    ----------
+    progress: `bool`
+        Show a progress bar during download
+    overwrite: `bool`
+        If exist overwrites the downloaded sample data.
+    timeout: `float`
+        The timeout in seconds. If `None` the default timeout is used from
+        `astropy.utils.data.Conf.remote_timeout`.
+    Returns
+    -------
+    None
+    """
+
+    print(sampledata_dir)
+    number_of_files_fetched = 0
+    print("Downloading sample files to {}".format(sampledata_dir))
+    for handle, file_name in _files.items():
```

Comment:
If ```_files``` and ```_base_urls``` are variables defined at the top of the module for use in subsequent functions without further definition, should they not be capitalised?  It took me a while to figure out how they were defined.